frontend/src/components/BtoDetailsPage.tsx

```diff
@@ -67,7 +70,10 @@ export default function BtoDetailsPage({btoDisplayItems, changeBtoItem, username
                         <ShowDummyText btoDisplayItem={btoItem} username={username}/> : <></>
                     }
                     {btoItem.status === "VOTE" ?
-                        <Vote btoItem={btoItem} username={username} changeBtoItem={changeBtoItem}/>
+                        <div>
+                            <h1>Vote getroffen</h1>
```

Comment:
Why "Vote" and not "Wahl"? The whole app is german...

Reply:
Eigentlich ist der Programmfluss und Variablen in Englisch nur die Benutzerschnittstelle sollte Deutsch sein.